sqrl-tools/sqrl-cli/src/main/java/com/datasqrl/cmd/AbstractCompilerCommand.java

```diff
@@ -84,6 +121,128 @@ public void runCommand(ErrorCollector errors) {
     }
   }
 
+  private boolean isGenerateGraphql() {
```

Comment:
Should all these live in the AbstractCompilerCommand?

Reply:
This is the command that handles the switch so it'd make sense that we'd check it here.